include/nbl/asset/filters/kernels/CConvolutionWeightFunction.h

```diff
@@ -84,7 +84,7 @@ class CConvolutionWeightFunction1D final : public impl::IWeightFunction1D<typena
 		const WeightFunction1DA m_funcA;
 		const WeightFunction1DB m_funcB;
 
-		value_t weight_impl(const float x, const uint32_t sampleCount) const
+		value_t NBL_API2 weight_impl(const float x, const uint32_t sampleCount) const
```

Comment:
funny it needs an NBL_API2, I thought it would be inline, can you explicitly declare it inline instead and see what happens?